index.js

```diff
@@ -506,13 +507,23 @@ class GithubScm extends Scm {
         // For pull requests
         if (config.prRef) {
             const prRef = config.prRef.replace('merge', 'head:pr');
+            const prNumber = prRef.split('/')[1];
+
+            // eslint-disable-next-line no-useless-escape
+            const regex = '\'s\/^\\[{\\(.*\\)}]$\/\\1\/\'';
 
             // Fetch a pull request
             command.push(`echo Fetching PR and merging with ${branch}`);
             command.push(`$SD_GIT_WRAPPER "git fetch origin ${prRef}"`);
+
+            // eslint-disable-next-line no-useless-escape
+            // eslint-disable-next-line max-len
+            command.push(`curl -s https://api.${config.host}/repos/${config.org}/${config.repo}/pulls/${prNumber} | sed ${regex} | tr ',' '\n' | grep -m1 'ref' | cut -f 2 -d ':' | tr -d '\"' | tr -d ' ' > prBranchName.json`);
```

Comment:
I want to use the `jq` command.
However, the end time of the step becomes longer.